app/shared/navbar/navbar.html

```diff
@@ -1,6 +1,6 @@
 <nav class="mb-1 navbar navbar-expand-lg navbar-light navbar-background orange lighten-1 sticky-top">
-    <a class="navbar-brand" href="#"><i class="fas fa-shopping-cart"></i>
-        <span class="badge badge-notify rounded-circle text-light">3</span></a>
+    <a class="navbar-brand" href="cart"><i class="fas fa-shopping-cart"></i>
```

Comment:
recommended to use ng-href instead of href

Reply:
ok, I replaced every angular attribute to data-ng-*